soh/soh/Enhancements/randomizer/3drando/playthrough.cpp

```diff
@@ -58,7 +59,7 @@ int Playthrough_Init(uint32_t seed, std::unordered_map<RandomizerSettingKey, uin
     GenerateHash();
     WriteIngameSpoilerLog();
 
-    if (Settings::GenerateSpoilerLog) {
+    if (/*Settings::GenerateSpoilerLog TODO: do we ever not want to write a spoiler log?*/ true) {
```

Comment:
Race mode seeds seem like a good time not to do it, just like how we also lock down all the settings with those.